.github/workflows/dockerfile-health.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: "Dockerfile Health"
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - .devcontainer/Dockerfile
+      - .github/workflows/dockerfile-health.yml
+      - lte/gateway/docker/mme/Dockerfile.ubuntu20.04
```

Comment:
Should we do the same for the ubuntu18? dockerfile? Or even better, can we converge to the 20.04 one?

Reply:
I was going to get this in place and confirm it's good - and then ask @rdefosse if he wants me to cover the ubuntu 18 dockerfile. I suspect he already has coverage over in the MAGMA-OAI-MME land?  As to convergence, that would depend upon the OAI use cases for the Dockerfile.ubuntu18.04 builds.